packages/xstate-test/test/plans.test.ts

```diff
@@ -67,7 +68,7 @@ describe('testModel.testPlans(...)', () => {
       }
     });
 
-    await testModel.testPlans(plans);
+    await testUtils.testModel(testModel);
```

Comment:
A model isn't what's tested; it's plans and paths that are tested. I feel this can be replaced in the future by:

```js
await testModel.getPlans().test();
```